### DISCUSSION TITLE
Publish/Hide pages/groups, not only spaces

### DISCUSSION THREAD

**Discussion by Gomilu** [GitbookIO/community#22] [Feature Requests]
### What's your feature/idea?

What do you want propose to GitBook?

Able to hide/publish pages, or groups within a space. Currently, AFAIK, only possible to hide/publish a space.

### What's your desired outcome?

What would you like to be able to do, if your idea came to life?

Able to edit pages while hidden. Then publish them whenever I desire. And then Hide again.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

I am new to change requests / merging. Maybe this is simply a change request + Merge and no need to implement that feature.

**Comment by taszap** [comment]
Hi @Gomilu although you are right that you can use change request as a workaround where your page/article remains in draft, it is not a full fledged solution. 

 This request is linked to #12 in some aspects.

**Comment by zachshallbetter** [comment]
I'd be satisfied with the ability to hide an item in the sidebar via a "menu item" 
I imagine the best method to exclude incomplete work would be via a feature branch. However, I'm not sure if this is something we can do currently.
